packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -509,7 +588,196 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
                 </oui-checkbox-description>
             </oui-checkbox>
         </div>
-        <div class="row mb-3">
+        <div
+            data-ng-if="$ctrl.isLocalPrivateMode() && $ctrl.isLocalZone() && !$ctrl.isAddingPrivateNetwork"
+        >
+            <oui-checkbox
+                class="mb-3"
+                data-model="$ctrl.isLocalPrivateModeLocalZone"
+            >
+                <oui-checkbox-label>
+                    <span
+                        data-translate="pci_projects_project_instances_add_privateNetwork_local_private_mode"
+                    ></span>
+                    <span class="oui-badge oui-badge_promotion my-1">
```

Comment:
**question:** Shouldn't the closing `span` come here?
```suggestion
                    <span class="oui-badge oui-badge_promotion my-1"></span>
```

Reply:
this is the tag localzone if we change the closing span we lose the correct style for the localzone tag